OSError: "<pending_xref py:class=\"DatalakeRESTException\" py:module=\"azure.datalake.store.exceptions\"\
  \ refdoc=\"azure.datalake.store\" refdomain=\"py\" refexplicit=\"False\" reftarget=\"\
  OSError\" reftype=\"class\" refwarn=\"False\">\n    <literal classes=\"xref py py-class\"\
  >\n        OSError\n"
object: "<pending_xref py:class=\"CountUpDownLatch\" py:module=\"azure.datalake.store.utils\"\
  \ refdoc=\"azure.datalake.store\" refdomain=\"py\" refexplicit=\"False\" reftarget=\"\
  object\" reftype=\"class\" refwarn=\"False\">\n    <literal classes=\"xref py py-class\"\
  >\n        object\n"
tuple: "<pending_xref py:class=\"File\" py:module=\"azure.datalake.store.transfer\"\
  \ refdoc=\"azure.datalake.store\" refdomain=\"py\" refexplicit=\"False\" reftarget=\"\
  tuple\" reftype=\"class\" refwarn=\"False\">\n    <literal classes=\"xref py py-class\"\
  >\n        tuple\n"
